### PR TITLE
Fixes, and makes the Sec Ushanka Better.

### LINKED ISSUE #12693: Sec Ushanka Reverts to Normal Ushanka when toggled.
Reporting client version: 514.1581

## Round ID:
5026

## Testmerges:
- [YOU CURRENTLY NEED THIS PR TO RUN THE SERVER WITH A DATABASE CONNECTED ](https://github.com/Skyrat-SS13/Skyrat-tg/pull/10823)
- [[NON-MODULAR] Small MODsuits remaster](https://github.com/Skyrat-SS13/Skyrat-tg/pull/12310)

## Reproduction:
Take Sec Ushanka.
Flip sec ushanka.
Turns into russian ushanka.

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

Title.
image
image

## How This Contributes To The Skyrat Roleplay Experience

The sprite is better. It doesn't turn into a soviet one.
Closes https://github.com/Skyrat-SS13/Skyrat-tg/issues/12693
## Changelog

:cl: Cursor (published the PR) Certain Unnamed TG Sprite Maintainer (done most of the sprite) Halcyon (done the folded back) and Plateau (Started the sprite recolour, done the coding.)
fix: The Sec Ushanka doesn't revert to a Soviet one.
imageadd: Made the Sec Ushanka not ugly.
/:cl:

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
epic

File: modular_skyrat/modules/customization/modules/clothing/head/head.dm [hunk 1]
```diff
@@ -123,13 +123,22 @@
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
 	icon_state = "ushankared"
 	inhand_icon_state = "ushankadown"
+	mutant_variants = NONE
```
Comment (theselfish) [inline modular_skyrat/modules/customization/modules/clothing/head/head.dm]:
```suggestion
```

File: modular_skyrat/modules/customization/modules/clothing/head/head.dm [hunk 2]
```diff
@@ -123,13 +123,22 @@
...
 
+
 /obj/item/clothing/head/ushanka/sec/blue
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	desc = "There's more to life than money, with this blue ushanka, you can prove it for $19.99."
 	icon_state = "ushankablue"
 	inhand_icon_state = "ushankadown"
+
+	mutant_variants = NONE
```
Comment (theselfish) [inline modular_skyrat/modules/customization/modules/clothing/head/head.dm]:
```suggestion
```

**Comment by theselfish** [comment]
@Gandalf2k15 Missed the mutant varient. Fixed now.

File: modular_skyrat/modules/customization/modules/clothing/head/head.dm [hunk 3]
```diff
@@ -123,13 +123,20 @@
...
 	supports_variations_flags = NONE
 
+
 /obj/item/clothing/head/ushanka/sec/blue
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	desc = "There's more to life than money, with this blue ushanka, you can prove it for $19.99."
 	icon_state = "ushankablue"
 	inhand_icon_state = "ushankadown"
+
```
Comment (theselfish) [inline modular_skyrat/modules/customization/modules/clothing/head/head.dm]:
```suggestion
```

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**